test/run_pass/test_console.js

```diff
@@ -13,9 +13,6 @@
  * limitations under the License.
  */
 
-/*
-  @STDOUT=FILE[test_console_stdout.txt]
```

Comment:
@LaszloLango Using js driver, I don't have a good idea to check out to compare console output with the real output which js made.